modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,182 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="etcd-increase-db_{context}"]
+= Increasing the database size for etcd
+
+You can set the disk quota in gigabytes for each etcd instance. If you set a disk quota for your etcd instance, you can specify integer values from 8 to 32. The default value is 8. You can specify only increasing values.
+
+:FeatureName: Increasing the database size for etcd
+include::snippets/technology-preview.adoc[]
+
+[id="etcd-change-db-size_{context}"]
+== Changing the etcd database size
+
+To change the database size for etcd, complete the following steps.
+
+.Procedure
+
+. Check the current value of the disk quota for each etcd instance by entering the following command:
++
+[source,terminal]
+----
+$ oc describe etcd/cluster | grep "Backend Quota"
+----
++
+.Example output
+[source,terminal]
+----
+Backend Quota Gi B: <value>
+----
+
+. Change the value of the disk quota by entering the following command:
++
+[source,terminal]
+----
+$ oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": <value>}}'
+----
++
+.Example output
+[source,terminal]
+----
+etcd.operator.openshift.io/cluster patched
+----
+
+.Verification
+
+. Verify that the new value for the disk quota is set by entering the following command:
++
+[source,terminal]
+----
+$ oc describe etcd/cluster | grep "Backend Quota"
+----
++
+The etcd Operator automatically rolls out the etcd instances with the new values. 
+
+. Verify that the etcd pods are up and running by entering the following command:
```

Comment:
`oc get pods -n openshift-etcd`
That'll show all of the etcd-related pods; specifically looking for the "4/4 Running" for each of the masters.

They could instead add a watch that would show the rollout happening instead of the instantaneous output of the above command.
`oc get pods -n openshift-etcd -w`

Whichever you think is better.